commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,32 @@
      }
 
     /**
+     * Calculates the page number of the result.
+     *
+     * @return the page number of the result, 0 if the page size is 0, meaning "{@code getCount()}" returns 0.
+     */
+    default Long getPage() {
+
+        if(getCount() == 0)
+            return 0L;
+
+        return getOffset() / getCount();
+    }
+
+    /**
+     * Calculates the total number of pages matching the request.
+     *
+     * @return the total number of pages , 0 if the page size is 0, meaning "{@code getCount()}" returns 0.
+     */
+    default Long getPagesCount() {
```

Comment:
I also was suffering figuring out a name, but lets consider `getTotalPages` since it seem pretty clear.